Source/sound.cpp

```diff
@@ -59,7 +59,7 @@ void CleanupMusic()
 		sgnMusicTrack = NUM_MUSIC;
 #ifdef DISABLE_STREAMING_MUSIC
 	if (musicBuffer != nullptr) {
-		mem_free_dbg(musicBuffer);
+		delete[] musicBuffer;
 		musicBuffer = nullptr;

```

Comment:
Ditto on the unique_ptr stuff.

Reply:
There isn't much difference with this one as we're still managing it manually

Reply:
It's still cleaner.

Reply:
I would find it a bit confusing